src/main/java/org/cbioportal/cdd/repository/topbraid/ClinicalAttributeMetadataRepositoryTopBraidImpl.java

```diff
@@ -0,0 +1,105 @@
+        "PREFIX skos:<http://www.w3.org/2004/02/skos/core#> " +
+        "SELECT DISTINCT ?study_id ?column_header (SAMPLE(?PriorityValue) AS ?priority) (SAMPLE(?AttributeTypeValue) AS ?attribute_type) (SAMPLE(?DatatypeValue) AS ?datatype) (SAMPLE(?DescriptionValue) AS ?description) (SAMPLE(?DisplayNameValue) AS ?display_name) " +
+        "WHERE { " +
+        "    GRAPH <urn:x-evn-master:clinical_data_dictionary> { " +
+        "        ?node rdf:type ?type. " +
+        "        ?node skos:broader ?parent. " +
+        "        ?parent cdd:StudyId ?study_id. " +
+        "        ?parent skos:broader ?grandparent. " +
+        "        ?grandparent rdfs:label ?column_header. " +
+        "        OPTIONAL{?node cdd:PriorityValue ?PriorityValue}. " +
+        "        OPTIONAL{?node cdd:AttributeTypeValue ?AttributeTypeValue}. " +
+        "        OPTIONAL{?node cdd:DatatypeValue ?DatatypeValue}. " +
+        "        OPTIONAL{?node cdd:DescriptionValue ?DescriptionValue}. " +
+        "        OPTIONAL{?node cdd:DisplayNameValue ?DisplayNameValue}. " +
+        "    } " +
+        "} " +
+        "GROUP BY ?study_id ?column_header " +
+        "ORDER BY ?study_id ?column_header " +
+        "VALUES ?type {cdd:ClinicalAttributeOverridePriorityValue cdd:ClinicalAttributeOverrideAttributeTypeValue cdd:ClinicalAttributeOverrideDatatypeValue cdd:ClinicalAttributeOverrideDescriptionValue cdd:ClinicalAttributeOverrideDisplayNameValue}";
+
+    private String queryClinicalAttributes = "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#> " +
+        "PREFIX cdd:<http://data.mskcc.org/ontologies/clinical_data_dictionary/> " +
+        "PREFIX skos:<http://www.w3.org/2004/02/skos/core#> " +
+        "SELECT ?column_header ?display_name ?attribute_type ?datatype ?description ?priority " +
+        "WHERE { " +
+        "    GRAPH <urn:x-evn-master:clinical_data_dictionary> { " +
+        "        ?subject rdfs:label ?column_header. " +
+        "        ?subject cdd:AttributeType ?attribute_type. " +
+        "        ?subject cdd:Datatype ?datatype. " +
+        "        ?subject cdd:Description ?description. " +
+        "        ?subject cdd:DisplayName ?display_name. " +
+        "        ?subject cdd:Priority ?priority. " +
+        "    } " +
+        "}";
+
+    public List<ClinicalAttributeMetadata> getClinicalAttributeMetadata() {
+        try {
+            return super.query(queryClinicalAttributes, new ParameterizedTypeReference<List<ClinicalAttributeMetadata>>(){});
+        } catch (TopBraidException e) {
+            logger.error("Problem connecting to TopBraid");
+            throw new RuntimeException(e);
+        }
+    }
+
+    public Map<String, ArrayList<ClinicalAttributeMetadata>> getClinicalAttributeMetadataOverrides() {
+        try {
+            List<ClinicalAttributeMetadata> overridesList = super.query(queryOverrides, new ParameterizedTypeReference<List<ClinicalAttributeMetadata>>(){});
```

Comment:
I would use a derived type here .. to make it clear when we needed the StudyId and when we did not. But that can be a future refactor.